chain/starknet/src/codec.rs

```diff
@@ -0,0 +1,32 @@
+#[allow(clippy::all)]
+#[rustfmt::skip]
+#[path = "protobuf/zklend.starknet.r#type.v1.rs"]
+pub mod pbcodec;
```

Comment:
I think this could be private since we are already exporting the types a few lines below. (which is preferable IMO)

Reply:
Hmmm was following the `near` counterpart haha. Sure I will make it private.

Reply:
Made private.